pkg/client/parse.go

```diff
@@ -88,21 +89,27 @@ func parseReport(filePath string, tags string, verbose bool) ([]fern.SuiteRun, e
 	return suiteRuns, err
 }
 
-func parseTestSuite(testSuite junit.TestSuite, tags string, verbose bool) (suiteRun fern.SuiteRun, err error) {
+func parseTestSuite(clock util.Clock, testSuite junit.TestSuite, tags string, verbose bool) (suiteRun fern.SuiteRun, err error) {
 	if verbose {
 		log.Default().Printf("Parsing TestSuite %s\n", testSuite.Name)
 	}
 
 	suiteRun.SuiteName = testSuite.Name
 
-	suiteRun.StartTime, err = time.Parse(time.RFC3339, testSuite.Timestamp)
-	if err != nil {
-		suiteRun.StartTime, err = time.Parse(time.RFC3339, testSuite.Timestamp+"Z")
+	if testSuite.Timestamp == "" {
+		suiteRun.StartTime = clock.Now()
+	} else {
+		suiteRun.StartTime, err = time.Parse(time.RFC3339, testSuite.Timestamp)
 		if err != nil {
-			err = fmt.Errorf("failed to parse suite start time: %w", err)
-			return
+			// Attempt to parse with a "Z" suffix for UTC time if the initial parsing fails
+			suiteRun.StartTime, err = time.Parse(time.RFC3339, testSuite.Timestamp+"Z")
+			if err != nil {
+				err = fmt.Errorf("failed to parse suite start time: %w", err)
+				return
+			}
```

Comment:
Should we allow the format of the time in timestamps to be configurable via a flag?

Reply:
i dont think so as of right now.